<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerWidget</class>
 <widget class="QWidget" name="TimerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLCDNumber" name="lcdMinutes">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="lcdSeconds">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="lcdMiliseconds">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
